covid-nlp/main.ipynb

```diff
@@ -0,0 +1,2384 @@
+{
```

Comment:
<p>done.</p><p>about that If you can, along the pipeline blocks, bellow them show an example sentence and how it would progress through the pipeline that would be awsome, we have that in the samples texts</p>

Reply:
<p>Its good in the sample texts, but the image you put in the presentation makes the illustration more clear, please put that version here, with the running example.</p>